@here/harp-mapview/lib/MapView.ts

```diff
@@ -1511,7 +1529,97 @@ export class MapView extends THREE.EventDispatcher {
             );
         }
 
-        this.update();
+        this.updateCamera();
+    }
+
+    /**
+     * Returns the camera focus point position in geo-coordinates. If the camera points to an
+     * invalid position will return `null`.
+     */
+    get geoTarget() {
```

Comment:
I would rather just name this `target`. Did you check the confluence page where we discussed the naming?